source/loader/ur_libapi.cpp

```diff
@@ -7743,4 +7743,8 @@ ur_result_t UR_APICALL urUsmP2PPeerAccessGetInfoExp(
     return exceptionToResult(std::current_exception());
 }
 
+void uselessFunc() {
```

Comment:
Since this file is autogenerated you need to add it to the mako file https://github.com/oneapi-src/unified-runtime/blob/main/scripts/templates/libapi.cpp.mako. You will then need to run the generator with `ninja generate` and commit all the changes.

Reply:
More info https://oneapi-src.github.io/unified-runtime/core/CONTRIB.html#generating-source if you need

Reply:
Is this OK now? As far as I can tell the `ur_lib.hpp` isn't autogenerated as well?